CHANGELOG.md

```diff
@@ -8,6 +8,102 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+| [Country Risk](https://github.com/eclipse-tractusx/vas-country-risk) | country-risk: [4.0.2](https://github.com/eclipse-tractusx/vas-country-risk/releases/tag/country-risk-4.0.2) | [1.3.1](https://github.com/eclipse-tractusx/vas-country-risk/releases/tag/v1.3.1) |
+
+### Known Knowns
+
+- Relevant components and interfaces developed considering Gaia-X Trust Framework - 22.10
+- Connection to an external Clearing House service for the entire Catena-X ecosystem is required. The Gaia-X compliance service should generally be capable of recognizing all companies that are able to register with the Catena-X Portal.
+Conformity to [Asset Administration Shell (AAS) API v3.0](https://industrialdigitaltwin.org/wp-content/uploads/2023/04/IDTA-01002-3-0_SpecificationAssetAdministrationShell_Part2_API.pdf) and bugfix v3.0.1 is given. Fine granular access control introduced for optional use.
+- This Tractus-X release does not include a Managed Identity Wallet (MIW). The development team decided on a temporary bridging solution, based on a commercial application, to test and ship Release 24.08. Without implementing this interim solution, participants will not be able to fully utilize this Tractus-X release in consequence.
+- For E2E Testing, the [1.2.0](https://github.com/SAP/ssi-dim-middle-layer/releases/tag/v1.2.0) of the [SSI DIM Middle Layer](https://github.com/SAP/ssi-dim-middle-layer) was used - this component is FOSS but the currently used wallet (see previous item) is not.
+- The Tractus-X EDC has undergone significant changes in its Management API, communication with external services and internal APIs.
+- Documentation content (e.g. architecture, manuals etc) and folder structure have not explicitly been checked for this Release, as relevant TRGs are not yet in force. The vast majority of components are evolutions of previous Releases, with a thorough review undergone. This leads to a high confidence in the quality of existing documentation within the product repositories.
+- FOSS components developed and pre-tested to TRL6 with artificial test data only
+- No crosscheck of functionality in verification environment
+- No execution of Load-, Performance- or Penetration Tests
+- No assignment of Export Control Classification Numbers (ECCN) to FOSS components
+- GeoBlocking recommended for Operations (GBaaS)
+- Security concept (overall) recommended for Operations
+- Listed components in "Unchanged, untested" category were not verified in combination with the current Release package and are only compatible with Catena-X Standards as specified. Use at your own risk at this time.
+
+### Runtime
+
+The release has been tested
+
+- on [Kubernetes](https://en.wikipedia.org/wiki/Kubernetes) versions: `1.28.9`
+- with [PostgreSQL](https://en.wikipedia.org/wiki/PostgreSQL) versions: `15`
+
+Note: Tractus-X EDC has been tested on both Postgresql versions: 15.x and 16.x
+
+The following violations of TRG 5.07 apply:
```

Comment:
was this copy/paste from some previous release? or are those point still valid?
@nicoprow could you please provide the info? Thanks.

Reply:
@evegufy at least from my side it was just copy&paste from 24.05. I already asked everybody (via direct mail) and also mailing list to go through the known knowns. 

But apart from your feedback, there weren't that many points. @ds-mwesener told me, that he collected some points.

Reply:
BPDM is not in violation of TRG 5.07 anymore. The newest release uses the approved Postgres Helm chart and appversions.

Reply:
Edc still tested with both 15.x and 16.x version, same as before
integration space of knowledge agents uses 16.x version

Reply:
@tom-rm-meyer-ISST Not sure if i did it correct, but i deleted the violation on bpdm side. Is there a need to change something related to postgres? What about country risk violation?

Reply:
Country Risk has been untested. Thus, good question. We could just put a note: "For known knowns of untested software please refer to previous changelog" OR just keep it and append "(untested)"

Being honest regarding BPDM @nicoprow @stephanbcbauer, it has been tested on int like that. We explain that but commonly the **tested** postgres is relevant
![image](https://github.com/user-attachments/assets/cce22668-f09d-4cf8-8e85-39a7415a339f)

Reply:
I agree with @tom-rm-meyer-ISST and think and a note like the following is the most proper way: "For known knowns of unchanged/untested components, please refer to previous changelog(s)"

Reply:
> Country Risk has been untested. Thus, good question. We could just put a note: "For known knowns of untested software please refer to previous changelog" OR just keep it and append "(untested)"
> 
> Being honest regarding BPDM @nicoprow @stephanbcbauer, it has been tested on int like that. We explain that but commonly the **tested** postgres is relevant. Not sure why there are two different postgres with different versions? In the end I trust on your knowledge of how it has been tested :) ![image](https://private-user-images.githubusercontent.com/116013795/354480972-cce22668-f09d-4cf8-8e85-39a7415a339f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI1ODMwMzUsIm5iZiI6MTcyMjU4MjczNSwicGF0aCI6Ii8xMTYwMTM3OTUvMzU0NDgwOTcyLWNjZTIyNjY4LWYwOWQtNGNmOC04ZTg1LTM5YTc0MTVhMzM5Zi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODAyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgwMlQwNzEyMTVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kMGRlNDQ1ZjUzMzgyZTRmMTMyOTJlMWY1ZDFhYjZmZGZhYTg0ZjcyN2IyNGMzNzdkNDA3MGQ4ZTZiMzY0MjhlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.hfFUOmocOe8QGLfT0ztfQEVI7jmC0SdxTPuXNNhsFFY)

Also, regarding Country Risk: yes it is untested and can stay like that

Reply:
added the note for known knowns with this [73ee8f7](https://github.com/eclipse-tractusx/tractus-x-release/pull/34/commits/73ee8f7898d49aae044d16f394a1295d0323af93) THX